library/clinical_rules.php

```diff
@@ -388,7 +386,7 @@ function compare_log_alerts($patient_id, $current_targets, $category = 'clinical
 {
 
     if (empty($userid)) {
-        $userid = $_SESSION['authId'];
+        $userid = $_SESSION['authUserID'];
```

Comment:
Was the authId to authUserID changes to fix a bug?

Reply:
this was changed in 503 so I changed here. why do we have an authID and a authUserID in sessions? Are they different meanings or an oversight?

Reply:
now i remember. Those were redundant session variables (both were user id), so we removed the `authId` one in 600 (no longer 503; gonna take awhile to get used to :) )